src/App.jsx

```diff
@@ -2,17 +2,18 @@ import Header from "./components/header";
 import { Stack, ThemeProvider } from "@mui/material";
 import theme from "./themes/theme";
 import { BrowserRouter } from "react-router-dom";
-import Footer from './components/footer';
+import Footer from "./components/footer";
 import Banner from "./pages/home/banner-section";
 
+import Footer from "./components/footer";
+import Home from "./pages/home";
 
 function App() {
   return (
     <BrowserRouter>
       <ThemeProvider theme={theme}>
-        <Header />
-        <Banner/>
-        <Footer/>
+        <Home />
```

Comment:
I see here you removed the Header and Banner components, Would you please remove their imports too.